src/FluentResults/Factories/Results.cs

```diff
@@ -53,6 +53,19 @@ public static Result Fail(string errorMessage)
             return result;
         }
         
+        /// <summary>
+        /// Creates a failed result with the given error messages. Internally a list of error objects from type <see cref="Error"/> is created
+        /// </summary>
+        public static Result Fail(IEnumerable<string> errorMessages)
+        {
+            if (errorMessages == null)
+                throw new ArgumentNullException(nameof(errorMessages), "The list of error messages cannot be null");
+            
+            var result = new Result();
+            result.WithErrors(Settings.MultipleErrorFactory(errorMessages));
```

Comment:
Please don't introduce a new setting - we can use the existing Settings.ErrorFactory and call it here in a foreach loop or something like ```errorMessage.Select(e => Settings.ErrorFactory(e))```. If we introduce a MultiErrorFactory then we have to set the same behavior twice if we want to overwrite the default error factory